### DISCUSSION TITLE
owner tag

### DISCUSSION THREAD

**Discussion by atfam** [Scythe-Anticheat/Scythe-Anticheat#149] [Ideas]
Hey it's me

I was wondering if you can make it a owner tag

I can do:
If the player has that tag They cannot be banned and One player at a time can only have

@MrDiamond64

**Comment by MrDiamond64** [comment]
Does the answer I gave at https://github.com/Scythe-Anticheat/Scythe-Anticheat/issues/144#issuecomment-2192685286 not do this?
